### ISSUE TITLE
大佬，发现一个小问题，3-full_sft.py中 加载预训练权重的代码被注释掉了，所以可能导致有其他大佬微调之后结果也不太好。

### ISSUE THREAD

**Issue by sunxinti** [jingyaogong/minimind#32]
if model_from == 1:
        model = Transformer(lm_config)
        # 加载预训练权重的代码被注释掉了
        # ckp = f'./out/pretrain_{lm_config.dim}{moe_path}.pth'
        # state_dict = torch.load(ckp, map_location=device)
        # unwanted_prefix = '_orig_mod.'
        # for k, v in list(state_dict.items()):
        #     if k.startswith(unwanted_prefix):
        #         state_dict[k[len(unwanted_prefix):]] = state_dict.pop(k)
        # model.load_state_dict(state_dict, strict=False)
    else:
        model = AutoModel.from_pretrained('./minimind', trust_remote_code=True)

**Comment by jingyaogong** [comment]
这个是怕很多人没有对应的.pth权重文件就直接做SFT，应该不会有人把这里给漏掉吧😂

还是解开注释吧~
